airflow/dags/dag_test.py

```diff
@@ -0,0 +1,65 @@
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+from airflow.utils.dates import days_ago
+
+
+my_dag = DAG(
+    dag_id='my_dag_of_the_morning',
+    description='My DAG to know what to do in the morning',
+    tags=['tutorial', 'datascientest'],
+    schedule_interval=None,
+    default_args={
+        'owner': 'airflow',
+        'start_date': days_ago(2),
```

Comment:
specify start_date at dag level in constructor instead of task level in default_args